backend/app/core/service.py

```diff
@@ -0,0 +1,64 @@
+from fastapi import FastAPI
+from fastapi.responses import JSONResponse
+from fastapi.security import HTTPBearer
+from fastapi_mail import ConnectionConfig, FastMail, MessageSchema, MessageType
+from pydantic import EmailStr
+
+from app.core.config import MAIL_FROM, MAIL_PASSWORD, MAIL_USERNAME
+from app.core.security import generate_verification_token
+
+conf = ConnectionConfig(
+    MAIL_USERNAME=MAIL_USERNAME,
+    MAIL_PASSWORD=MAIL_PASSWORD,
+    MAIL_FROM=MAIL_FROM,
+    MAIL_PORT=587,
+    MAIL_SERVER="smtp.gmail.com",
+    MAIL_STARTTLS=True,
+    MAIL_SSL_TLS=False,
+    USE_CREDENTIALS=True,
+    VALIDATE_CERTS=True,
+)
+
+
+app = FastAPI()
+
+
+bearer_scheme = HTTPBearer()
+
+
+ALGORITHM = "HS256"
+ACCESS_TOKEN_EXPIRE_MINUTES = 300
+TOKEN_EXPIRE_MINUTES = 30
+
+
+async def simple_send(email: EmailStr, token: str) -> JSONResponse:
+    verify_url = "http://127.0.0.1:8000/api/v1/auth/verify"
+    button_html = f"""
+    <form method="post" action="{verify_url}">
+        <input type="hidden" name="email" value="{email}">
+        <input type="hidden" name="v_token" value="{token}">
+        <button type="submit" style="background-color: #008CBA; border: none; color: white; padding: 15px 32px; text-align: center; text-decoration: none; display: inline-block; font-size: 16px; margin: 4px 2px; cursor: pointer;">Verify Email</button>
+    </form>
+    """
```

Comment:
Add this form to the template folder as HTML file.

Reply:
Created separate template files for each html code snippet in the templates folder:

```
<!DOCTYPE html>
<html>
<head>
    <style>
        .button {
            background-color: #008CBA;
            border: none;
            color: white;
            padding: 15px 32px;
            text-align: center;
            text-decoration: none;
            display: inline-block;
            font-size: 16px;
            margin: 4px 2px;
            cursor: pointer;
        }
    </style>
</head>
<body>
    <h3>This Token will be valid for 30 minutes only.</h3>
    <br>
    <p>Click to Verify:</p>
    <form method="post" action="{{ verify_url }}">
        <input type="hidden" name="email" value="{{ email }}">
        <input type="hidden" name="v_token" value="{{ token }}">
        <button type="submit" class="button">Verify Email</button>
    </form>
</body>
</html>

```